hardhat.config.ts

```diff
@@ -110,6 +110,10 @@ export default {
       ...sharedNetworkConfig,
       url: `https://linea-goerli.infura.io/v3/${INFURA_KEY}`,
     },
+    linea: {
+      ...sharedNetworkConfig,
+      url: `https://linea.infura.io/v3/${INFURA_KEY}`,
```

Comment:
Linea also supplied a default RPC 

```suggestion
      url: "https://rpc.linea.build",
```